pandas/tests/indexes/test_numeric.py

```diff
@@ -1187,3 +1187,39 @@ def test_range_float_union_dtype():
 
     result = other.union(index)
     tm.assert_index_equal(result, expected)
+
+
+def test_uint64_keys_in_list():
+    # https://github.com/pandas-dev/pandas/issues/28023
+    bug = pd.Series(
```

Comment:
Could both these test functions be combined into 1 func? Seem very similar

Reply:
They are indeed similar. I'll wait for a few more comments to decide on this.

Reply:
yea pls do we prefer parametized tests as much as possible

Reply:
Revisiting the issues, I realized one of the tests would be redundant and removed that.

Reply:
> Revisiting the issues, I realized one of the tests would be redundant and removed that.

How come? - it seems that test case was exactly covering issue #28023

Reply:
I would just use https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.testing.assert_index_equal.html

Reply:
#28023 reports a `KeyError`. If https://github.com/pandas-dev/pandas/pull/29420/files#diff-7c97f2c17a9193409668a6f698905c73R1206 holds, KeyError won't be raised :)

Reply:
About the test function... Now that I'm rooting for a single test, I'll leave it as `isinstance` since I think the intention of test is clearer this way.